rqt_image_view/src/rqt_image_view/image_view.cpp

```diff
@@ -77,11 +77,20 @@ void ImageView::initPlugin(qt_gui_cpp::PluginContext& context)
 
   ui_.save_as_image_push_button->setIcon(QIcon::fromTheme("image"));
   connect(ui_.save_as_image_push_button, SIGNAL(pressed()), this, SLOT(saveImage()));
+
+  pub_topic_custom_ = false;
```

Comment:
Why do we need a boolean variable for this? The state seems to be always available in the line edit widget.